docs/dictionary/property/currentCard.lcdoc

```diff
@@ -0,0 +1,34 @@
+Name: currentCard
+Type: property
+
+Introduced: 2.7.2
+
+Syntax: 
+get the currentCard of <stack>
+set the currentCard of <stack> to <cardName>
+
+Value: short name of a card in a stack
+
+Summary:
+The short name of the <current card> of a stack
+
+Associations: stack
+
+OS: mac, windows, linux, ios, android
+Platforms: desktop, server, mobile
+
+Example:
+--The following results in the same value as put the short name of the current card of this stack
+ put the currentCard of this stack
+--This property is useful if the stack is invisible, not in the foreground or offscreen
+set the visible of this stack to false
+put the currentCard of this stack
+--The currentCard property can be retrieved or set for a stack that is not currently an opened file
+set currentCard of stack "/Users/admin/Desktop/example.livecode" to "ExampleOfCardNamedFoo"
+
+Description:
+Use the <currentCard> property to change the <current card> of a stack, or find out which card is the <current card> of a stack. The taget stack does not need to be the frontmost stack, visible and the stack file need need not be open (file write access is needed in order to set the property).
```

Comment:
Most things should be wrapped at 72 characters now. This was scripted a few weeks back. Code wasn't touched because it is harder to hard wrap and syntax doesn't support wrapping. In general it's just links that overrun in other places.